app/views/shared/_loss_report_modal.html.erb

```diff
@@ -0,0 +1,19 @@
+<div id="loss-report-modal" class="eui-modal-content" >
+  <a href="javascript:void(0);" class="modal-close float-r" id="close-loss-report-modal"><i class="fa fa-times"></i><span class="is-invisible">Close</span></a>
+  <div>
+    <h4>Conversion to UMM Format Required</h4>
+    <p>The native format of this collection is <%= options[:native_format].split('/').last.gsub('+xml','').upcase %>. Editing this record using the MMT will convert it to UMM-JSON, which may result in data loss. Select 'View Loss Report' to see how the conversion will affect this record.</p>
+    <p style="display: inline-flex">
+      <% if current_provider?(options[:provider_id]) %>
+        <%= link_to 'View Loss Report', loss_report_collections_path(options[:concept_id], format: 'json'), class: 'button-spacing eui-btn--steel-blue', target: :_blank %>
+        <%= link_to 'Edit Collection', edit_collection_path(revision_id: options[:revision_id]), class: 'eui-btn--blue modal-close-link button-spacing' %>
+        <%= link_to 'Cancel', 'javascript:void(0);', class: 'eui-btn modal-close-link' %>
+      <% elsif available_provider?(options[:provider_id]) %>
+        <%= link_to 'View Loss Report', loss_report_collections_path(options[:concept_id], format: 'json'), class: 'button-spacing eui-btn--steel-blue', target: :_blank %>
+        <%= link_to 'Edit Collection', '#', class: 'eui-btn--blue loss-report button-spacing' %>
+        <%= link_to 'Cancel', 'javascript:void(0);', class: 'eui-btn modal-close-link' %>
+      <% end %>
+      <%= link_to 'Invisible Link', '#not-current-provider-modal', class: 'display-modal not-current-provider eui-btn is-invisible', id: 'loss-report-to-edit-collection', data: { provider: options[:provider_id], record_action: 'edit-collection' } %>
```

Comment:
I think we figured out how to not need this

Reply:
This instance is different than the `modal-close-link`. this causes the first modal to close and the next to open. reference coffeescript

Reply:
Couldn't the wrong provider modal move into the available_provider edit collection link?

Reply:
That was my first thought, but closing one modal (the `loss-report-modal`) then opening another (the `not-current-provider-modal`) requires coffeescript as far as I can tell

Reply:
Also simply adding the classes from 'invisible link' to the Edit link unfortunately doesn't work

Reply:
`    <%= link_to 'Edit Collection', '#not-current-provider-modal', class: 'eui-btn--blue loss-report button-spacing display-modal not-current-provider', data: { provider: options[:provider_id], record_action: 'edit-collection' } %>
    `
and removing the invisible link seems to work for me.  Should check if all of that is necessary since my check was really coarse.  Also lets you remove the click from the coffee, the hide is sufficient.

Reply:
I stand corrected, I did not properly add the classes from the invisible link to the Edit link. Thank you both for this suggestion